sdv/sequential/par.py

```diff
@@ -112,8 +112,10 @@ def _update_context_column_dict(self, context_columns_dict):
         default_transformers_by_sdtype = deepcopy(self._data_processor._transformers_by_sdtype)
         table_metadata = self.metadata.tables[self._table_name]
         for column in self.context_columns:
-            column_metadata = table_metadata.columns[column]
-            if default_transformers_by_sdtype.get(column_metadata['sdtype']):
+            sdtype = table_metadata.columns[column]['sdtype']
+            if sdtype == 'id':
+                context_columns_dict[column] = {'sdtype': 'categorical'}
```

Comment:
The issue states to generate value from scratch using the provided regex format. With this change, do we re-use values from the real data?